plugins/window-state/src/lib.rs

```diff
@@ -278,7 +278,7 @@ impl<R: Runtime> WindowExtInternal for WebviewWindow<R> {
 
 impl<R: Runtime> WindowExtInternal for Window<R> {
     fn update_state(&self, state: &mut WindowState, flags: StateFlags) -> tauri::Result<()> {
-        let is_maximized = match flags.intersects(StateFlags::MAXIMIZED | StateFlags::SIZE) {
+        let is_maximized = match flags.intersects(StateFlags::MAXIMIZED) {
```

Comment:
after taking a deeper look unfortunately I think this impacts https://github.com/tauri-apps/plugins-workspace/blob/5ec310199f9bf9a8b314ef2c6c1f8ff62c66ad22/plugins/window-state/src/lib.rs#L310 
this was been around since forever, I think we should instead fix the problem on the tao side, maybe return false if it's not a NSWindow?

Reply:
Hmm that's true. We can do what you suggested.

Reply:
I will close this PR then and open a new PR on tao.